<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAgregarCompraBase</class>
 <widget class="QWidget" name="FormAgregarCompraBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Datos de compra</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="LProveedor">
          <property name="text">
           <string>Proveedor:</string>
          </property>
          <property name="buddy">
           <cstring>CBProveedor</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ECBProveedor" name="CBProveedor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
          <property name="duplicatesEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="LFormaPago">
          <property name="text">
           <string>Forma de pago:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBContado">
          <property name="text">
           <string>Contado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBOtro">
          <property name="text">
           <string>Otro</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="LFecha">
          <property name="text">
           <string>Fecha:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="DEFecha">
          <property name="currentSection">
           <enum>QDateTimeEdit::DaySection</enum>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ECBProveedor" name="CBProveedorProducto">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ECBProductos" name="CBProducto">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SBCant">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PBAgregarProducto">
       <property name="text">
        <string>Agregar Producto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTableView" name="TVLista"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="PBEliminarProducto">
         <property name="text">
          <string>Eliminar Producto</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ECBProductos</class>
   <extends>QComboBox</extends>
   <header>ecbproductos.h</header>
  </customwidget>
  <customwidget>
   <class>ECBProveedor</class>
   <extends>QComboBox</extends>
   <header>ecbproveedor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
